tests/node_converters/lrn_test.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import onnx
+
+from tests.utils.common import check_onnx_model
+from tests.utils.common import make_model_from_nodes
+
+
+def _test_lrn(data: np.ndarray, alpha: float, beta: float, bias: float, size: int) -> None:
+    test_inputs = {'input_tensor': data}
+    node = onnx.helper.make_node(
+        op_type='LRN',
+        inputs=list(test_inputs),
+        outputs=['y'],
+        alpha=alpha,  # ONNX attributes are passed as regular keyword arguments.
+        beta=beta,
+        bias=bias,
+        size=size,
+    )
+
+    model = make_model_from_nodes(
+        nodes=node,
+        initializers={},
+        inputs_example=test_inputs,
+    )
+    check_onnx_model(model, test_inputs)
+
+
+def test_lrn() -> None:  # pylint: disable=missing-function-docstring
+    shape = (227, 227, 3)
+    data = np.random.random_sample(shape).astype(np.float32)
+    alpha = np.random.uniform(low=0.0, high=1.0).astype(np.float32)
```

Comment:
You don't need to convert `float` to `np.float32`, just rm `.astype(np.float32)`.
This is only needed for arrays (when you pass size to uniform).

```python
>>> import numpy as np
>>> alpha = np.random.uniform(low=0.0, high=1.0)
>>> type(alpha)
<class 'float'>
```

```python
>>> alpha = np.random.uniform(low=0.0, high=1.0).astype(np.float32)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'float' object has no attribute 'astype'
```

Reply:
thank you for clarifying! Will update.